.circleci/config.yml

```diff
@@ -16,6 +16,9 @@ jobs:
       - checkout
       - restore_cache: { key: 'gradle-wrapper-{{ checksum "gradle/wrapper/gradle-wrapper.properties" }}' }
       - restore_cache: { key: 'gradle-cache-{{ checksum "versions.props" }}-{{ checksum "build.gradle" }}' }
+      - run: sudo apt-get update
+      - run: sudo apt-get install python-setuptools python-pip
+      - run: sudo pip install twine
```

Comment:
These lines diverge from the .circleci/config.yml template, so they'll be overwritten by excavator.  We'd need to brainstorm a slightly different way of doing python integration tests, or possibly upstream something into the template.

Reply:
i'm unfamiliar with this templating/excavator system. Where does the template live and how is it controlled?